src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,45 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_STORAGE = 10;
+    private K[] keys;
+    private V[] values;
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```

Default int values is 0